.dockerignore

```diff
@@ -0,0 +1 @@
+fly.toml
```

Comment:
Actually I just realized after the assignment that it's good to include `.git` here: I'd suggest adding that.